supervisor/plugins/cli.py

```diff
@@ -52,7 +52,9 @@ async def load(self) -> None:
 
             await self.instance.attach(version=self.version)
         except DockerError:
-            _LOGGER.info("No cli plugin Docker image %s found.", self.instance.image)
+            raise DockerError(
```

Comment:
This should not be converted, we do not want to raise here.

Reply:
Okay I will do it .